src/components/OrgSettings/General/GeneralSettings.tsx

```diff
@@ -19,36 +19,46 @@ interface InterfaceGeneralSettingsProps {
  * @param props - The properties passed to the component.
  * @returns The `GeneralSettings` component.
  */
+
 const GeneralSettings: FC<InterfaceGeneralSettingsProps> = ({ orgId }) => {
   const { t } = useTranslation('translation', {
     keyPrefix: 'orgSettings',
   });
 
   return (
     <Row className={`${styles.settingsBody} mt-3`}>
-      <Col lg={7}>
-        <Card className="rounded-4 mb-4 mx-auto shadow-sm border border-light-subtle">
-          <div className={styles.cardHeader}>
-            <div className={styles.cardTitle}>{t('updateOrganization')}</div>
-          </div>
+      <Col xxl={7} xl={12} className="mb-4">
+        <Card
+          className={`rounded-4 mb-4 shadow-sm border border-light-subtle ${styles.mainCard}`}
+        >
+          <Card.Header className={styles.deleteCardHeader}>
+            <h5 className={`mb-0 fw-semibold ${styles.cardHeading} `}>
+              {t('Edit Organization')}
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.